docs/Installation.md

```diff
@@ -16,6 +16,11 @@ Build Support_ component when installing Unity.
        width="500" border="10" />
 </p>
 
+## Windows installation
+For setting up your environment on Windows, we have created a [detailed
+guide](Installation-Windows.md) to setting up your env. For Mac and Linux,
+continue with this guide.
+
 ## Clone the ML-Agents Toolkit Repository
```

Comment:
This header should be renamed to "## Mac and Unix Users", immediately follows by a sub-heading "### Clone the ML-Agents Toolkit Repository"

Reply:
Further, below:
- `## Install Python and mlagents Package` --> `### Install Python and mlagents Package`
- `### NOTES` --> `**Notes:**` (bold instead of header)
- Delete `### Mac and Unix Users` header

These suggested are following the thinking that the intention here is to have three top-level sections:
- Windows Users
- Mac and Unix Users
- Docker Installation